swift/TirePressure/Sources/TirePressure/Alarm.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public class Alarm {
+    private let lowPressureThreshold: Double = 17
+    private let highPressureThreshold: Double = 21
+
+    let _sensor = Sensor()
```

Comment:
I think _sensor needs to be private like the other variables. In some language versions it's not, but I think it should be

Reply:
True! I've updated the line
Thanks